src/pages/ProfilePage/profileAPI.js

```diff
@@ -0,0 +1,24 @@
+import { testnetContractAddress } from '../../common/contansts';
+
+const options = {
+  method: 'GET',
+  headers: { accept: 'application/json', 'x-api-key': 'GjhopFFeeF5GM9egesqoTNzmwHUTReQ4ZgZD3RKe' },
+};
+
+const api = {
+
+  fetchData: async (address) => {
+    try {
+      const response = await fetch(`https://api-testnet.starkscan.co/api/v0/nfts?contract_address=${testnetContractAddress}&owner_address=${address}`, options);
```

Comment:
since we'll have different APIs and keys for starkscan, again keep this as part of config

Reply:
Done